EditorPluginExample/typings/Atomic/EditorWork.d.ts

```diff
@@ -7,6 +7,7 @@
 
 /// <reference path="Atomic.d.ts" />
 /// <reference path="Editor.d.ts" />
+/// <reference path="ToolCore.d.ts" />
```

Comment:
I concur, need to consolidate and version *.d.ts files and address the *Work.d.ts files.  "typings" in project folders should maybe be an internal folder which the editor populates with current ambient definitions or something, or which our tsc compiler includes from a standardized editor/cli folder